.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: ci_pipeline
```

Comment:
I think you can make the name translate what Is being done in the workflow, like `"Linting and Testing"`, for example. What do you think?